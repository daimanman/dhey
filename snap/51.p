{"C":1,"Duration":63670601267,"EndTime":"2018-08-23T14:07:47.6625616+08:00","Err":"","FileId":"4","Method":"POST","N":1,"P":{"info":"TEssssssssskkkkkk颠三倒四ST","name":"sssssDXwwM"},"Payload":"","QH":{},"Remark":"压力测试","ReqNums":1,"StartTime":"2018-08-23T14:07:47.6545818+08:00","Status":1,"TaskId":51,"Type":"FIXED","Url":"http://localhost:9188/go1","Z":""}