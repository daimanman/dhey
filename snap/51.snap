{"AvgTotal":0.0079798,"Fastest":0.0079798,"Slowest":0.0079798,"Average":0.0079798,"Rps":125.31642397052558,"AvgConn":0.0039581,"AvgDNS":0.0039581,"AvgReq":0,"AvgRes":0.0007833,"AvgDelay":0.0022096,"ConnMax":0.0039581,"ConnMin":0.0039581,"DnsMax":0.0039581,"DnsMin":0.0039581,"ReqMin":0,"ReqMax":0,"ResMin":0,"ResMax":0.0007833,"DelayMax":0.0022096,"DelayMin":0.0022096,"Lats":[0.0079798],"ConnLats":[0.0039581],"DnsLats":[0.0039581],"ReqLats":[0],"ResLats":[0.0007833],"DelayLats":[0.0022096],"Total":7979800,"ErrorDist":{},"StatusCodeDist":{"200":1},"SizeTotal":62,"SizeReq":62,"NumRes":1,"LatencyDistribution":[{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0}],"Histogram":[{"Mark":0.0079798,"Count":1,"Frequentcy":1},{"Mark":0.0079798,"Count":0,"Frequentcy":0},{"Mark":0.0079798,"Count":0,"Frequentcy":0},{"Mark":0.0079798,"Count":0,"Frequentcy":0},{"Mark":0.0079798,"Count":0,"Frequentcy":0},{"Mark":0.0079798,"Count":0,"Frequentcy":0},{"Mark":0.0079798,"Count":0,"Frequentcy":0},{"Mark":0.0079798,"Count":0,"Frequentcy":0},{"Mark":0.0079798,"Count":0,"Frequentcy":0},{"Mark":0.0079798,"Count":0,"Frequentcy":0},{"Mark":0.0079798,"Count":0,"Frequentcy":0}],"TaskId":51,"Duration":0.0079798}